clients/web/src/views/widgets/GroupAdminsWidget.js

```diff
@@ -21,46 +21,43 @@ var GroupAdminsWidget = View.extend({
         'click .g-demote-moderator': function (e) {
             var li = $(e.currentTarget).parents('.g-group-admins>li');
             var userid = li.attr('userid');
-            var view = this;
 
             confirm({
                 text: 'Are you sure you want to demote <b>' +
                     _.escape(li.attr('username')) + '</b> to a moderator?',
                 escapedHtml: true,
-                confirmCallback: function () {
+                confirmCallback: () => {
                     var user = new UserModel({_id: userid});
-                    view.model.off('g:promoted').on('g:promoted', function () {
+                    this.model.off('g:promoted').on('g:promoted', () => {
                         this.trigger('g:moderatorAdded');
-                    }, view).promoteUser(user, AccessType.WRITE);
```

Comment:
Removing the context binding passed to `on` actually changes the behavior, since unbinding is often done using that context reference. It may not matter here for lifecycle issues, but I'd be more comfortable leaving the context variable (though it can be changed to `this`).